distributions/aws/test/e2e/utils/config.py

```diff
@@ -51,7 +51,7 @@ def to_file(self):
         filepath = os.path.abspath(os.path.join(METADATA_FOLDER, filename))
 
         with safe_open(filepath, "w") as file:
-            json.dump(self.params, file)
+            json.dump(self.params, file, indent=4)
```

Comment:
why did we need this level of indentation ?

Reply:
helps with readability if browsing/modifying the file

Reply:
can we also add .json to the file name so IDEs are able to recognise them?

Reply:
sure